src/tabs/tabs.js

```diff
@@ -84,7 +84,7 @@ export default class Tabs extends React.Component<TabsPropsT> {
           {...tabContentProps}
           {...props}
         >
-          {isActive ? child.props.children : null}
+          {child.props.children}
```

Comment:
It would be nice for performance to optionally not render children. Or rather the other way around where it defaults to the current functionality, and will apply this seo benefit based on a boolean prop.

Reply:
Where there is a `boolean` attribute passed at the `Tabs` level? Or the `Tab` level potentially?
I could add something like `renderAll` as a prop on `Tabs` for now. What do you think @chasestarr?

Reply:
I would rather make it global on the container `Tabs` component. Could you see a use-case for doing so individually?

Reply:
Depends on the amount of tabs or scenario. You could potentially have some tab where it's critical that its content is rendered but other tabs not so much. It's an edge case that may be worth considering in the future 👍